### PR TITLE
Profile fix for OJT estimates.

### PR DESCRIPTION
**Pull request by mphprogrammer**
Code Review
Addresses the following Issues: 

1. Estimator values not displayed for OJT type schools.
2. Malformed tags affecting layout and logic.
3. Creation of component partials to simply code.

Affected files: profile_overview.html.erb, profile_components/*.html.erb

To test - enter a search term bringing back several schools (e.g. "New York"), filter on Employer to bring up OJT schools, compare any OJT estimator values in search results box to those that are displayed on the profile when the search link is clicked.

### CODE REVIEW

File: spec/tasks/build_db_spec.rb [hunk 1]
```diff
@@ -1,70 +1,70 @@
...
+#    let(:all_valid_floats) { 3 }
+#    let(:no_valid_floats) { 4 }
+#    let(:valid_float_value) { 1234.5 }
+
+#    it "valid strings are floats" do
+#      row = Institution.find_by(facility_code: @csv_rows[all_valid_floats][:facility_code])
+#      @floats.each { |float| puts "@@@@@@@@@@@@ #{float}"; expect(row[float]).to eq(valid_float_value) }
+#      end
+
+#    it "invalid strings are 0" do
```
Comment (plusjeff) [inline spec/tasks/build_db_spec.rb]: Any reason why all these commented out lines can't be deleted (and recovered if need be via Git)

File: app/views/institutions/_profile_overview.html.erb [hunk 2]
```diff
@@ -1,310 +1,106 @@
...
-	<% end %>
-</div>
-
-<!-- END Veteran Summary -->
-
-</div>
-
-<!-- GI Bill Students -->
-<div id="vetgroups" class="modal-style clearfix" style="display: none; position: fixed; opacity: 1; z-index: 11000; left: 50%; margin-left: -202px; top: 200px;">
+    <%= render partial: 'institutions/profile_components/caution_flags' %>
```
Comment (plusjeff) [inline app/views/institutions/_profile_overview.html.erb]: :heart: the switch to partials. Makes this page's ERB much cleaner!

**Comment by plusjeff** [comment]
Got an error going to one of the pages.

[profile-bug.mp4.zip](https://github.com/department-of-veterans-affairs/gi-bill-comparison-tool/files/128749/profile-bug.mp4.zip)

image

**Comment by plusjeff** [comment]
Assigning back for fixes/comments. Assign back to me when ready to go.

**Comment by mphprogrammer** [comment]
Weird I can't duplicate it. I'll look into it.

**Comment by mphprogrammer** [comment]
How did you get it?

**Comment by plusjeff** [comment]
See the attached video (link just above the screenshot)

**Comment by mphprogrammer** [comment]
OK Found a long standing issue with partial path resolution. Moved profile_components directory to /app/views. If this doesn't do it I'll eat a bug....

Err.... No, I won't.

**Pull request merged**